pkgs/tools/audio/botamusique/default.nix

```
@@ -0,0 +1,136 @@
+{ pkgs,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3Packages,
+  coreutils,
+  nix-prefetch-git,
+  jq,
+  nodePackages
+}:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, python3Packages
, coreutils
, nix-prefetch-git
, jq
, nodePackages
}:
```
Generally pkgs shouldn't be used as an input.

---

`pkgs` is taken by `node-env.nix`, so can't get around it unfortunately